targets/csharp-unityv2/source/Shared/Internal/PlayFabHttp/IPlayFabHttp.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace PlayFab.Internal
+{
+
+    public interface IPlayFabHttp
+    {
+        void Awake();
```

Comment:
We should call this InitializeHttp() to match the function in PlayFabHttp which calls it.
Awake falsely implies that it's related to a MonoBehaviour, which is no longer true.